inventory/sample/group_vars/k8s-cluster.yml

```diff
@@ -246,3 +246,16 @@ persistent_volumes_enabled: false
 ## See https://github.com/kubernetes-incubator/kubespray/issues/2141
 ## Set this variable to true to get rid of this issue
 volume_cross_zone_attachment: false
+
+## Container Engine Acceleration
+## Enable container accelartion feature, for example use gpu acceleration in containers
+# nvidia_accelerator_enabled: true
+## Nvidia GPU driver install. Install will by done by a (init) pod running as a daemonset.
+## Array with nvida_gpu_nodes, leave empty or comment if you dont't want to install drivers.
```

Comment:
Can I have a label that I define in my inventory file that would install the gpu stuff?

Reply:
Yes, you can the driver will install, but there is also a taint set on the nodes in `nvidia_gpu_nodes` which prevents scheduling of non gpu pods on gpu nodes. That is why this array exsists. 

If you would add the label `nvidia.com/gpu=true` via inventory  node and have `nvidia_accelerator_enabled` then the driver will be installed but then the taint won't be set.